docs/190-paas-operations/003-backup-and-restore.md

```diff
@@ -2,7 +2,7 @@
 layout: cluedin
 nav_order: 3
 parent: PaaS operations
-permalink: /deployment/ama-backup
+permalink: /paas-operations/ama-backup
```

Comment:
Since you changed the permalink here, please update the reference in the file "050-ama-fourth-step.md", section "Set up a backup solution".